CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.23
+
+###### _Jan 14, 2021_
+
+Big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Support public paths in module augmentation (#24267) @eps1lon
+- 👩‍🎤 Migrate the Paper to emotion (#24397) @povilass
+- 📚 Implemented various CI improvements (#24313) @eps1lon, (#24358) @oliviertassinari
```

Comment:
We don't mention dev ops improvements unless have an obvious first order value for the developers reading the changelog highlight. I think that we should remove this line. 

```suggestion
```